setup.py

```diff
@@ -23,5 +23,6 @@
         'requests',
         'beautifulsoup4',
         'selenium',
+        'lxml',
```

Comment:
Where is `lxml` used?

Reply:
`lxml` is used for parsing the RSS feed using BeautifulSoup:
```
feed_soup = BeautifulSoup(feed_contents, features='xml')
```